bip-schnorr.mediawiki

```diff
@@ -147,10 +147,15 @@ The algorithm ''Sign(sk, m)'' is defined as:
 * Let ''e = int(hash<sub>BIPSchnorr</sub>(bytes(R) || bytes(P) || m)) mod n''.
 * Return the signature ''bytes(R) || bytes((k + ed) mod n)''.
 
-'''Above deterministic derivation of ''R'' is designed specifically for this signing algorithm and may not be secure when used in other signature schemes.'''
```

Comment:
Given that people already made this mistake it's perhaps better to keep the warning as conspicuous as it was.